### ISSUE TITLE
[Accessibility] ARIA / Screen reader

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt#2403]
- [x] Ensure all visible items have a title or an aria-label describing them
- [x] Dropdown field, use aria-expanded=true/false. Make sure it works with a screen reader. 
- [x] All icons should have aria-hidden=true, role=presentation
- [x] Make sure all menu items have role="menuitem"
- [x] Make sure all divs with role="complementary" have a unique label
- [x] For tabs, use role=tablist and role=tabpanel, aria-controls and aria-labeledby to link everything
- [x] Search box (read out number of returned items)

**Comment by veler** [comment]
All those point are part of [https://github.com/Microsoft/pxt/pull/2446](https://github.com/Microsoft/pxt/pull/2446)
